lib/l10n/app_tr.arb

```diff
@@ -0,0 +1,638 @@
+  "entryInfoActionEditRating": "Derecelendirmeyi düzenle",
+  "entryInfoActionEditTags": "Etiketleri düzenle",
+  "entryInfoActionRemoveMetadata": "Meta verileri kaldır",
+
+  "filterBinLabel": "Geri dönüşüm kutusu",
+  "filterFavouriteLabel": "Favori",
+  "filterLocationEmptyLabel": "Konumsuz",
+  "filterTagEmptyLabel": "Etiketsiz",
+  "filterRatingUnratedLabel": "Derecelendirilmemiş",
+  "filterRatingRejectedLabel": "Reddedilmiş",
+  "filterTypeAnimatedLabel": "Hareketli",
+  "filterTypeMotionPhotoLabel": "Hareketli Fotoğraf",
+  "filterTypePanoramaLabel": "Panorama",
+  "filterTypeRawLabel": "Raw",
+  "filterTypeSphericalVideoLabel": "360° Video",
+  "filterTypeGeotiffLabel": "GeoTIFF",
+  "filterMimeImageLabel": "Resim",
+  "filterMimeVideoLabel": "Video",
+
+  "coordinateFormatDms": "DMS",
+  "coordinateFormatDecimal": "Ondalık dereceler",
+  "coordinateDms": "{coordinate} {direction}",
+
+  "coordinateDmsNorth": "K",
+  "coordinateDmsSouth": "G",
+  "coordinateDmsEast": "D",
+  "coordinateDmsWest": "B",
+
+  "unitSystemMetric": "Metrik",
+  "unitSystemImperial": "İngiliz",
+
+  "videoLoopModeNever": "Asla",
+  "videoLoopModeShortOnly": "Yalnızca kısa videolar",
+  "videoLoopModeAlways": "Her zaman",
+
+  "videoControlsPlay": "Oynat",
+  "videoControlsPlaySeek": "Oynat ve ileri/geri git",
+  "videoControlsPlayOutside": "Başka bir oynatıcı ile aç",
+  "videoControlsNone": "Hiçbiri",
+
+  "mapStyleGoogleNormal": "Google Haritalar",
+  "mapStyleGoogleHybrid": "Google Haritalar (Hibrit)",
+  "mapStyleGoogleTerrain": "Google Haritalar (Arazi)",
+  "mapStyleHuaweiNormal": "Petal Haritalar",
+  "mapStyleHuaweiTerrain": "Petal Haritalar (Arazi)",
+  "mapStyleOsmHot": "İnsancıl OSM",
+  "mapStyleStamenToner": "Stamen Tonik",
+  "mapStyleStamenWatercolor": "Stamen Suluboya",
+
+  "nameConflictStrategyRename": "Yeniden adlandır",
+  "nameConflictStrategyReplace": "Değiştir",
+  "nameConflictStrategySkip": "Atla",
+
+  "keepScreenOnNever": "Asla",
+  "keepScreenOnViewerOnly": "Yalnızca görüntüleyici sayfası",
+  "keepScreenOnAlways": "Her zaman",
+
+  "accessibilityAnimationsRemove": "Ekran efektlerini önle",
+  "accessibilityAnimationsKeep": "Ekran efektlerini koru",
+
+  "displayRefreshRatePreferHighest": "En yüksek oran",
+  "displayRefreshRatePreferLowest": "En düşük oran",
+
+  "themeBrightnessLight": "Açık",
+  "themeBrightnessDark": "Koyu",
+  "themeBrightnessBlack": "Siyah",
+
+  "albumTierNew": "Yeni",
+  "albumTierPinned": "Sabitlenmiş",
+  "albumTierSpecial": "Genel",
+  "albumTierApps": "Uygulamalar",
+  "albumTierRegular": "Diğer",
+
+  "storageVolumeDescriptionFallbackPrimary": "Dahili depolama",
+  "storageVolumeDescriptionFallbackNonPrimary": "SD kart",
+  "rootDirectoryDescription": "kök dizin",
+  "otherDirectoryDescription": "“{name}” dizin",
+
+  "storageAccessDialogTitle": "Depolama Erişimi",
+  "storageAccessDialogMessage": "Bu uygulamaya erişim sağlamak için lütfen bir sonraki ekranda "{volume}" {directory}'sini seçin.",
```

Comment:
Using double straight quotes `"` will brake the file format (and for typography's sake, we can do better). I'll replace them with `“...”` unless you do it first.

Reply:
```suggestion
  "storageAccessDialogMessage": "Bu uygulamaya erişim sağlamak için lütfen bir sonraki ekranda “{volume}” öğesinin {directory} dizinini seçin.",
```

Reply:
Changed, thanks.